docs/js.rst

```diff
@@ -30,11 +30,19 @@ with:
 
 .. code-block:: js
 
+<<<<<<< HEAD
```

Comment:
something is wrong here with merging my docs fixes from develop branch. <<HEAD and >>develop left here.